atlp-rwanda/champs-ec-fe
# Ft seller should be able to  create a product-#187300180

#### What does this PR do?
This PR introduces functionality for sellers to create new products within the e-commerce platform.

#### Description of Task to be completed?
This PR enhances the e-commerce platform to allow sellers to login. Upon successful login, the system verifies the user's role as a seller and directs them to a dedicated dashboard where they can create new products.

#### How should this be manually tested?
To test this feature manually:
- Clone this repository.
- Run `npm run dev`.
- Login with seller credentials and verify redirection to the seller dashboard.
- Navigate to the product creation page and ensure sellers can successfully add new products.

#### Any background context you want to provide?
This feature addresses user demand for sellers to independently manage their product listings, enhancing the platform's usability and empowering sellers.

#### What are the relevant pivotal tracker/Trello stories?
This PR matches with requirement on story with id #187300197 (https://www.pivotaltracker.com/n/projects/2698211/stories/187300197)

#### Screenshots (if appropriate)

![Screenshot 2024-06-18 084248](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/c2f4ae27-52bb-47be-84bc-03a56b20d9e4)

![Screenshot 2024-06-18 072853](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/f631167d-8b90-499f-a7c2-031cd70cd902)

![Screenshot 2024-06-14 093740](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/4cf0574f-d78e-47c5-b798-1295be19baed)
![Screenshot 2024-06-14 093809](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/d8939ae0-12a2-4d35-9234-5ced64c1abf2)
![Screenshot 2024-06-14 093834](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/a06915b8-eab5-4438-ac7f-d1d255538d2b)
![Screenshot 2024-06-14 093843](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/63a291cd-8f90-408f-b4b8-996e7c796721)
![Screenshot 2024-06-14 093911](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/3d134521-ae58-4d68-bc04-75699a930d18)
![Screenshot 2024-06-14 093930](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/9fb34d38-bc81-4a2f-ab2c-d35bf0d32337)
![Screenshot 2024-06-14 093938](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/e1641095-733d-4963-b270-a7ebc077b6f8)
![Screenshot 2024-06-14 093952](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/cc010c4c-b2a8-4501-8eec-32a344b1a564)
![Screenshot 2024-06-14 094049](https://github.com/atlp-rwanda/champs-ec-fe/assets/141851169/c4b94e40-1e54-41af-a411-b756f3954df8)
